src/lib/accounts/index.ts

```diff
@@ -0,0 +1,122 @@
+import {
+  AccountBase,
+  sign,
+  messageToHash,
+  RpcRejectedByUserError,
+  unpackTx,
+} from '@aeternity/aepp-sdk-13';
+import { Encoded } from '@aeternity/aepp-sdk-13/src/utils/encoder';
+import { Store } from 'vuex';
+import { useAccounts } from '../../composables/accounts';
+import { IS_CORDOVA, IS_EXTENSION_BACKGROUND } from '../environment';
+import { POPUP_TYPE_MESSAGE_SIGN, POPUP_TYPE_SIGN, POPUP_TYPE_TX_SIGN } from '../../popup/utils';
+import { showPopup } from '../../background/popupHandler';
+
+export class AccountSuperhero extends AccountBase {
+  address: Encoded.AccountAddress;
+
+  store: Store<any>;
+
+  aeppInfo: Record<string, any> = {};
+
+  constructor(store: Store<any>) {
+    super();
+    this.store = store;
+    const { account } = useAccounts({ store: this.store });
+    this.address = account.value.address as Encoded.AccountAddress;
+  }
+
+  signTransaction = (txBase64: Encoded.Transaction, opt: any): Promise<Encoded.Transaction> => {
```

Comment:
I suggest to define class methods as regular function (as described in the documentation: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes#class_body). In general it's more common.